packages/binding-coap/src/coaps-client.ts

```diff
@@ -109,7 +109,7 @@ export default class CoapsClient implements ProtocolClient {
             coaps.setSecurityParams(requestUri.hostname, this.authorization);
 
             coaps
-                .observe(form.href, "GET", next)
+                .observe(form.href, "get", next)
```

Comment:
At the moment it is somewhat confusing.

Some methods in the same file are CAPITALIIZED and lowerCased afterwards, some are lowercase only.
Not an issue of your update but I general concern I have.

Reply:
@danielpeintner Agree, I had the same feeling.